package.json

```diff
@@ -25,16 +25,16 @@
   },
   "homepage": "https://github.com/trentrand/cypress-utils#readme",
   "peerDependencies": {
-    "cypress": ">=5 <=8"
+    "cypress": ">=10"
   },
   "devDependencies": {
-    "cypress": "8.3.0"
+    "cypress": "10.7.0"
```

Comment:
maybe `^10.0.0` to support a wider range of users?

Reply:
Yes, sure.